openghg/retrieve/_access.py

```diff
@@ -1,20 +1,18 @@
 import json
 import logging
 from io import BytesIO
-from typing import Any, Dict, List, Optional, Union
-from openghg.types import SearchError
-import logging
-
-logger = logging.getLogger("openghg.retrieve.access")
```

Comment:
Did this line get removed?

Reply:
Yes, there is a line below it `logger = logging.getLogger("openghg.retrieve")` that redefines `logger`. I think the one I deleted was added by Elena at the code retreat. I guess the way we structure the imports might make it confusing to log messages from `openghg.retrieve.access`, but I suppose there was a reason for adding the more specific logger. (It's possible I didn't realise they were different and thought they were just duplicates.)

Reply:
Yes I see that in the unchanged bit now - I think leaving this as logging from `openghg.retrieve` is fine for now on this.